content/en/events/2023/kcsna/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Contributor Summit North America 2023"
+type: docs
+weight: 1
+description: >
+  Kubernetes Contributor Summit North America, hosted in Chicago, Illinois.
+
+---
+
+
+### About
+
+The North America Summit will be a lightweight event taking place on November 6th
```

Comment:
```suggestion
The North America summit will be a full contributor summit event taking place on November 6th
```
This is a full event as we have the CFP going this time. Not necessarily a blocker to this getting merged, but something we'd want to follow up on before blasting comms out.